GameOfRunes/GameOfRunes/GameEngine/GameMetaData.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 class GameMetaData {
     // TODO: maybe change this to current avatar.
     let avatar: Avatar
-    let stage: Stage
+    var stage: Stage
```

Comment:
Should this be immutable? Shouldnt additional waves be under `stageWaves` property?

Reply:
Yup. Forgot to change back!